File: PRAKTIKUM-PEMROGRAMAN1/MODUL4/C/PRAK405-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int n, k;
    scanf("%d %d", &n, &k);

    int total = 0;
    for (int i = 1; i <= n; i++) {
        int hasil = 0;
        for (int j = i; j >= 1; j--) {
            printf("(%d * %d)", j, k);
            hasil += j * k;
            if (j > 1) {
                printf(" + ");
            }
        }
        printf(" = %d\n", hasil);
        total += hasil;
    }
    printf("%d\n", total);
    return 0;
}

File: PRAKTIKUM-PEMROGRAMAN1/MODUL4/Python/PRAK405-2410817210024-MuhammadIbnuSina.py
n, k = map(int, input().split())

total = 0
for i in range(1, n + 1):
    hasil = 0
    for j in range(i, 0, -1):
        print(f"({j} * {k})", end="")
        hasil += j * k
        if j > 1:
            print(" + ", end="")
    print(f" = {hasil}")
    total += hasil

print(total)